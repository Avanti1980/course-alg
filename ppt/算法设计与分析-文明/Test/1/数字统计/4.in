5
3529332133840530986815473252108373206
13862755361737929257006769379171679844571964858793
308621824558744134495307297214824608983650444
20509059870791295875975230296902954069968950299787
64919150761158756662558599687326939045141459236913380350328253176