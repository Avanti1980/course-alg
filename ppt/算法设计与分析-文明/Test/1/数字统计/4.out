18202
16429
17716
176
6208